packages/terra-time-input/tests/jest/TimeInput.test.jsx

```diff
@@ -324,7 +324,7 @@ it('should validate the getVariantFromLocale helper method', () => {
   const wrapper = mountWithIntl(timeInput);
   const testComponent = wrapper.children();
 
-  if (['ar', 'en', 'en-AU', 'en-CA', 'en-GB', 'en-US', 'es', 'es-US'].includes(testComponent.instance().props.intl.locale)) {
+  if (['ar', 'en', 'en-AU', 'en-CA', 'en-GB', 'en-US', 'es-US'].includes(testComponent.instance().props.intl.locale)) {
```

Comment:
Can [this](https://github.com/cerner/terra-framework/blob/4597abe4c622d4d9ed83f7d3621e5168206bbd55/packages/terra-time-input/src/TimeInput.jsx#L158) also be updated now..?
```js
 if (TimeUtil.getVariantFromLocale(props) === TimeUtil.FORMAT_12_HOUR) {
    - if (!this.props.intl.messages['Terra.timeInput.am'] || !this.props.intl.messages['Terra.timeInput.pm']) {
       }
    }
```
To

```JS
 if (TimeUtil.getVariantFromLocale(props) !== TimeUtil.FORMAT_12_HOUR) {

 }
```

Reply:
Control was never going inside the condition so, removed it entirely and moved warning to separate condition f7bc706